vars/defaults.yml

```diff
@@ -13,6 +13,8 @@ main_user_shell: "/bin/bash"
 # encfs_password: (required)
 friendly_networks:
   - ""
+# letsencrypt_email: (required)
```

Comment:
I don't think email is required.

https://letsencrypt.readthedocs.org/en/latest/using.html#configuration-file